inc/mod/auth.php

```diff
@@ -6,6 +6,9 @@
 
 defined('TINYBOARD') or exit;
 
+//dont ask. auth was giving me a lot of stupid shit with already defined functions
+if(!function_exists("mkhash") && !function_exists("generate_salt") && !function_exists("login") && !function_exists("setCookies") && !function_exists("destroyCookies") && !function_exists("modLog") && !function_exists("create_pm_header") && !function_exists("make_secure_link_token") && !function_exists("check_login")){
```

Comment:
holy wat

i'll do this but i'll write the commit myself because there's no way i can commit this